eng/pipelines/runtimelab.yml

```diff
@@ -71,3 +71,11 @@ stages:
         bindings:
           isBindingsBuild: true
           scriptArgs: '--platform MacOSX --arch x86_64-apple-macos --framework StoreKit'
+
+    - template: /eng/pipelines/common/templates/test-job.yml
+      parameters:
+        osGroup: OSX
+        platform: maccatalyst
+        archType: x64
```

Comment:
Why x64? Is this a HW availability limitation?

Reply:
The `hostArchitecture: Arm64` was missing.

Reply:
not if you run tests directly on the AzDO build machine, the arm64 macOS pool is only available in internal builds, not in public.

Reply:
Is the arm64 build redundant then? Should we use Helix instead for arm64 containers?

Reply:
> which arm64 job? the one in this file only builds for arm64 which does work on x64.

Yes, that was my question. So even it builds for arm64, it runs x64.

Reply:
Yes